samples/msal-angular-v3-samples/angular15-sample-app/src/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Angular 15 Sample App</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
```

Comment:
@jo-arroyo we should track a cleanup item remove these, we shouldn't be using external fonts in our samples.  @derisen for FYI.  Thanks.

Reply:
Sounds good. Created.